### DISCUSSION TITLE
Inserting custom values into create resolver

### DISCUSSION THREAD

**Discussion by kash** [MichalLytek/typegraphql-prisma#205] [General]
I have a use case where I want to modify the data that goes into a table. The field that I want to set does not exist in the input type.

I want to set the `user_id` field for a table with the ID of the user that is currently logged it, which exists in `context`.

I've tried directly modifying the args.data object, but that doesn't seem to do the trick. This is what I've tried:

```
applyResolversEnhanceMap({
	PageView: {
		_all: [Authorized(Role.USER)],
		createPageView: [
			UseMiddleware(({args, context}, next) => {
				args.data.user_id = context.user.id;
				
				return next();
			})
		]
	},
});
```

My Prisma schema file:
```
model PageView {
  id String @id @default(uuid())
  
  /// @TypeGraphQL.omit(input: true)
  user_id    String
  /// @TypeGraphQL.omit(input: true)
  created_at DateTime @default(now())
}
```

Is there a way to do this without creating a custom resolver? I see myself doing something like this a lot. For example, hashing a password when a user is creating an account and using that value instead of the plain text that the user provides. Thanks!

**Comment by MichalLytek** [comment]
It's not possible yet, see #139 👀
